client/app/pages/settings/components/AuthSettings/SAMLSettings.jsx

```diff
@@ -9,28 +8,30 @@ import { SettingsEditorPropTypes, SettingsEditorDefaultProps } from "../prop-typ
 export default function SAMLSettings(props) {
   const { values, onChange } = props;
 
+  const onChangeEnabledStatus = e => {
+    const updates = { auth_saml_enabled: !!e.target.value };
+    if (e.target.value) {
+      updates.auth_saml_type = e.target.value;
+    }
+    onChange(updates);
+  };
+
   return (
     <DynamicComponent name="OrganizationSettings.SAMLSettings" {...props}>
       <h4>SAML</h4>
       <Form.Item label="SAML Enabled">
-        <Checkbox
-          name="auth_saml_enabled"
-          checked={values.auth_saml_enabled}
-          onChange={e => onChange({ auth_saml_enabled: e.target.checked })}>
-          SAML Enabled
-        </Checkbox>
+        <Radio.Group
+          onChange={onChangeEnabledStatus}
+          value={values.auth_saml_enabled && (values.auth_saml_type || "static")}>
```

Comment:
I replaced `defaultValue="static"` with this `|| "static"` because the first only targets `undefined` values and it seemed to come with `auth_saml_type=""` from the API.

Note: It is assumed that the upper value is handled as "static" from the backend side.

Reply:
This looks good, but we probably should change the default to "dynamic" instead to make migration more smooth. The way Redash did SSO before was "dynamic" by default. 

In the case that `auth_saml_enabled` is true and `auth_saml_type` is falsy, we'd assume that it was due to migration to v9, because there is no other organic way to get to that state, and in that case, we'd want "dynamic".

Reply:
Oh, right 😅. I mixed the defaults in my mind :) https://github.com/getredash/redash/pull/5175/files#r504974699

Reply:
I think I was the one who mixed them up 😅, thanks for fixing.